packages/solana/src/client.ts

```diff
@@ -235,6 +235,8 @@ export class Web3Modal extends Web3ModalScaffold {
       )
     )
 
+    OptionsController.state.enableSwaps = false
```

Comment:
No, we set the flags inside the `scaffold/client`. Please update this line conditionally: https://github.com/WalletConnect/web3modal/blob/main/packages/scaffold/src/client.ts#L355